db/migrations/03_create_users_table.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE users(
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(50),
+  email char(254) UNIQUE,
```

Comment:
I think email might need changing to VARCHAR, as char is a fixed length data type

Reply:
thanks Steph, I have made the changes as requested